pkg/daemon/ceph/client/filesystem.go

```diff
@@ -173,6 +174,11 @@ func IsMultiFSEnabled() bool {
 	}
 	return false
 }
+func SetPools(context *clusterd.Context, cephVersion cephver.CephVersion, clusterName, fsName string, metdataPool []*model.Pool, dataPool *model.Pool) error {
```

Comment:
Does this function really need to be exported?

Reply:
This function is defined here and used here https://github.com/rook/rook/pull/4433#discussion_r354334380